create_db.py

```diff
@@ -0,0 +1,11 @@
+from app import app, db
+
+
+def create_database():
```

Comment:
I think we should create the admistrator account here. According to the chosen method for the issue above:
- Use `ArgumentParser` to process firstname, surname and email for the admin account
- Use email and blank (first-)names values that will be updated at first login

Please also provide a short notice in the README how to run the app for the first time.